### DISCUSSION TITLE
How to know the version pytelegrambotapi?

### DISCUSSION THREAD

**Discussion by Criptons** [eternnoir/pyTelegramBotAPI#1592] [Q&A] [answered]
Hey there! I installed the library on ubuntu and wanted to find out which version pytelegrambotapi I have

**Comment by temirovazat** [comment, marked as answer]
If you have already installed you can try this:

```
from importlib.metadata import version
version('pyTelegramBotAPI')
```

or look in the list of dependencies:

`pip3 list`

**Reply by Criptons** [reply]
Thanks!
